### PR TITLE
Get rid of shell-out and invoke command directly via exec

### PR DESCRIPTION
**Pull request by StupidScience**
Hello.

Please let me know if this is something you expect here.
Or would be better to switch `execCommand` to something like `execCommand(ctx context.Context, mode ExecMode, commandName string, args ...string)`?

Related to #47

### CODE REVIEW

File: cmd/gotk/utils.go [hunk 1]
```diff
@@ -62,7 +62,12 @@ const (
 
 func (*Utils) execCommand(ctx context.Context, mode ExecMode, command string) (string, error) {
```
Comment (stefanprodan) [inline cmd/gotk/utils.go:63]: Can you please change the signature to:
```go
 execKubectlCommand(ctx context.Context, mode ExecMode, arg ...string) (string, error)
```

Comment (stefanprodan) [reply cmd/gotk/utils.go:63]: After the signature change we need to pass an args array, so the current string concatenation should be rewrite to array append.

Comment (StupidScience) [reply cmd/gotk/utils.go]: I've changed it

Comment (stefanprodan) [reply cmd/gotk/utils.go]: Can you please rebase with main.

Comment (StupidScience) [reply cmd/gotk/utils.go]: done

Comment (StupidScience) [reply cmd/gotk/utils.go]: Should I squash it or smth?

Comment (stefanprodan) [reply cmd/gotk/utils.go]: Yes please squash

**Comment by stefanprodan** [comment]
@StupidScience can you give this a try on Windows and see if it works? Thanks

**Comment by StupidScience** [comment]
Squashed. Also I tried to run `gotk check -pre` on Win10 and it successfully checked kubectl version. @stefanprodan

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @StupidScience 🎖️

**Pull request merged**